composer.json

```diff
@@ -3,10 +3,11 @@
     "type": "symfony-bundle",
     "description": "The missing Symfony Admin Generator",
     "keywords": [
-        "Admin Generator",
         "admin",
+        "bundle",
+        "php",
```

Comment:
What does this package have to do with PHP? It's not a parser…

Reply:
I tried to reproduce something similar to our existing topics: 
![image](https://user-images.githubusercontent.com/9052536/93670483-fbd2ea00-fa9b-11ea-8ddd-d4ab691bf4a9.png)

Reply:
Maybe the Github topic `php` can be added regardless of whether it appears here? I don't expect to obtain the same results when typing `php` in Github and in Packagist.

Reply:
Indeed, when I think about it everything on packagist is php related...

Reply:
Yeah, so on Packagist, I would expect `nikic/php-parser` or something like that to come up when I type `php`

Reply:
I removed php